plenum/server/notifier_plugin_manager.py

```diff
@@ -61,15 +61,26 @@ def sendMessageUponSuspiciousSpike(
         assert 'cnt' in historicalData
         assert 'minCnt' in config
         assert 'coefficient' in config
+        assert 'minActivityThreshold' in config
+        assert 'enabled' in config
+
+        if not config['enabled']:
+            logger.debug('Feature disabled')
```

Comment:
Can we provide a more detailed description on what is disabled?

Reply:
fixed